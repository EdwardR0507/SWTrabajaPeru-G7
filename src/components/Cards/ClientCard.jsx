import { React } from "react";
import { Divider, CardHeader, Avatar } from "@material-ui/core/";

const ClientCard = ({ nombres, imagen, servicio }) => {
  return (
    <>
      <CardHeader
        avatar={<Avatar>{imagen}</Avatar>}
        title={nombres}
        subheader={`Quiere solicitar el servicio de ${servicio}`}
      />

      <Divider />
    </>
  );
};
export default ClientCard;
