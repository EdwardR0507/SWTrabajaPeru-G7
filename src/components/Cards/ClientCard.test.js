/*Importamos las librerias principales*/
import React from "react";
import { shallow } from "enzyme";
import { MemoryRouter } from 'react-router-dom';
import { render, screen, fireEvent } from '@testing-library/react'
import ClientCard from "./ClientCard";
/*Renderizado del componente WorkerCard*/
describe("<ClientCard></ClientCard>", () => {
  it('render "Client"', () => {
    const wrapper = shallow(<ClientCard />);
    expect(wrapper).toMatchSnapshot()
  });
})