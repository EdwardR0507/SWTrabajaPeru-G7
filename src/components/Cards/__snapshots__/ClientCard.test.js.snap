// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClientCard></ClientCard> render "Client" 1`] = `ShallowWrapper {}`;
